.github/workflows/dotnetcore.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     - name: Setup .NET 8.0.x
       uses: actions/setup-dotnet@v2.1.0
       with:
-        dotnet-version: 8.0.100-rc.1.23463.5
+        dotnet-version: 8.0.100
```

Comment:
maybe 8.0.x? Not sure. we float patch for 7 but use a specific version for 6.

Reply:
8.0.x should work. I think on 6 we had to target a specific version at one time.

Reply:
nit: could you start from a new branch instead of merging w/main from a previous branch?

Reply:
donezo

Reply:
> nit: could you start from a new branch instead of merging w/main from a previous branch?

yes 😬